_includes/_masthead.html

```diff
@@ -49,13 +51,15 @@
 {% elsif page.header.image_fullwidth %}
 
 <div id="masthead">
+{% if site.logo %}
```

Comment:
Empty div here as well
(did you try if hiding these div messes up layout or not?)

Reply:
> This one is needed. Without it, you don't get the image.

Which image? only the logo is rendered inside. Is it JavaScript?

Reply:
The `page.header.image_fullwidth`. Yes, JavaScript. See line 11 of `_includes/_footer_scripts.html`

Reply:
k, thanks